### PR TITLE
Update QUEST_LV_0300.tsv

### PR DESCRIPTION
**Pull request by Satoram**
Some editing and review. This will need to be gone over again to make sure it is correct.

### CODE REVIEW

File: EnglishTranslation-master/QUEST_LV_0300.tsv [hunk 1]
```diff
@@ -291,7 +291,7 @@ QUEST_LV_0300_20151001_000290	To be able to read the letters, you must have a pi
 QUEST_LV_0300_20151001_000291	{nl}You can obtain the piece of the symbol somewhere in Galeed Plateau.
 QUEST_LV_0300_20151001_000292	(Something is written, but is not legible.)
 QUEST_LV_0300_20151001_000293	(It seems that something is written on it, but it can't be read in this condition.)
-QUEST_LV_0300_20151001_000294	That ring was given to me by my mother.{nl}I want to go back home with it, back to where my mother is...
+QUEST_LV_0300_20151001_000294	That ring was given to you by my mother.{nl}I want to go back home with it, back to where my mother is...
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0300.tsv]: Why the change? AFAIK, the mother has given her ring to this NPC.

File: EnglishTranslation-master/QUEST_LV_0300.tsv [hunk 2]
```diff
@@ -362,77 +362,77 @@ QUEST_LV_0300_20151001_000361	For Death (1)
...
-QUEST_LV_0300_20151001_000414	It seems that the poisonous plant isn't effective to demon Svitrigaila. Talk to him again.
+QUEST_LV_0300_20151001_000414	It seems that the poisonous plant isn't effective on demon Svitrigaila. Talk to him again.
 QUEST_LV_0300_20151001_000415	Demon Svitrigaila wants to die using the crystal since the poisonous plant isn't effective. Meet with the Cleric Master and obtain the divine water.
 QUEST_LV_0300_20151001_000416	For Death
 QUEST_LV_0300_20151001_000417	It seems that the demon, Svitrigaila already knows about it. Talk to him again what he should do.
 QUEST_LV_0300_20151001_000418	Destroy the barrier stone at Dvus Street
-QUEST_LV_0300_20151001_000419	The demon, Svitrigaila will die when you destroy the integrator using the barrier stone. Destroy the barrier stone that is protecting the integrator and collect the cores.
+QUEST_LV_0300_20151001_000419	The demon, Svitrigaila, will die when you destroy the integrator using the barrier stone. Destroy the barrier stone that is protecting the integrator and collect the cores.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0300.tsv]: The demon, Svitrigaila, -> Demon Svitrigaila

File: EnglishTranslation-master/QUEST_LV_0300.tsv [hunk 3]
```diff
@@ -461,27 +461,27 @@ QUEST_LV_0300_20151022_000460	This city... is very sad.{nl}I am guiding the spir
...
+QUEST_LV_0300_20151022_000472	You are a shameless swindler.{nl}I don't know why Justina's father has a friend like him.
 QUEST_LV_0300_20151022_000473	It is so fortunate that we found the family treasures of Justina.{nl}But, I should say more to that Pardoner!
 QUEST_LV_0300_20151022_000474	Who was that girl anyways.
-QUEST_LV_0300_20151022_000475	Be careful. Soon, the petrified frosts would blow. {nl}But, we don't know when...{nl}
+QUEST_LV_0300_20151022_000475	Be careful. Soon, the petrified frosts will blow. {nl}But, we don't know when...{nl}
 QUEST_LV_0300_20151022_000476	Is the detector at the supply route working alright?
 QUEST_LV_0300_20151022_000477	We may receive some help from Retia at Verkti Square.{nl}It's hard for us, Kupoles, to be related in human matters so I am counting on you.
-QUEST_LV_0300_20151022_000478	We've already given up one salvation, and we couldn't do anything on the Medzio Diena.{nl}We can't give up anymore.
+QUEST_LV_0300_20151022_000478	We've already given up one salvation, and we couldn't do anything on the day of Medzio Diena.{nl}We can't give up anymore.
```
Comment (Sunaries) [inline EnglishTranslation-master/QUEST_LV_0300.tsv]: on the day of Medzio Diena. -> on Medzio Diena.

**Comment by Sunaries** [comment]
Good to merge. :+1:

File: EnglishTranslation-master/QUEST_LV_0300.tsv [hunk 4]
```diff
@@ -362,77 +362,77 @@ QUEST_LV_0300_20151001_000361	For Death (1)
...
+QUEST_LV_0300_20151001_000371	Putting the orb in place
 QUEST_LV_0300_20151001_000372	Say Farewell
 QUEST_LV_0300_20151001_000373	Help
 QUEST_LV_0300_20151001_000374	Look at the petrification detector at the entrance of the Coastal Fortress
 QUEST_LV_0300_20151001_000375	Escape from the petrified frosts
-QUEST_LV_0300_20151001_000376	As you tried to look at the detector, the petrified frosts are blowing. Go to a safe place by avoiding petrified frosts!
-QUEST_LV_0300_20151001_000377	You were able to escape safely from petrified frosts due to Kupole Meile. Talk with Kupole Meile.
-QUEST_LV_0300_20151001_000378	In order for Kupole Meile to guide the spirits safely from petrified frosts, check all the detectors that you haven't been in this region.
-QUEST_LV_0300_20151001_000379	Tell the circumstances to Kupole Meile
+QUEST_LV_0300_20151001_000376	As you tried to look at the detector, the petrified frosts started to blow. Go to a safe place by avoiding petrified frosts!
```
Comment (ttgmichael) [inline EnglishTranslation-master/QUEST_LV_0300.tsv:376]: The petrified frosts blow against you as you tried to check the detector. Go to a safe place to avoid the petrified frosts!
